source/transactions/transactions.rst

```diff
@@ -810,26 +810,30 @@ certain retries of commitTransaction and abortTransaction) to the same mongos.
 When to unpin
```

Comment:
Can you bump the transaction spec and add a changelog entry?

Reply:
Done.